chain/client/src/view_client.rs

```diff
@@ -112,6 +114,9 @@ impl ViewClientRequestManager {
 }
 
 impl ViewClientActor {
+    /// Number of seconds between state requests. More frequent requests will be rejected.
+    const TIME_BETWEEN_STATE_REQUESTS: u64 = 30;
```

Comment:
```suggestion
    const TIME_BETWEEN_STATE_REQUESTS: Duration = Duration::from_secs(30);
```

It's better to call `as_secs` when using `TimedCache`.

I don't really understand why `TImedCache` is not using Duration already, this feels like a questionable API design, opened https://github.com/jaemk/cached/issues/74